bin/python_scripts/find_invalid_tags.py

```diff
@@ -70,20 +70,23 @@ def create_tag_list(rows):
 def create_invalid_tag_data(invalid_tags):
```

Comment:
I think it would be good to split the changes in this file into a separate commit.

Reply:
ok, will do